src/ui/public/courier/data_source/build_query/__tests__/from_filters.js

```diff
@@ -1,18 +1,12 @@
 import { buildQueryFromFilters } from '../from_filters';
```

Comment:
Could you just add a simple test here that verifies that the `query:queryString:options` aren't added in?